opennms-config/src/main/java/org/opennms/netmgt/config/PollerConfigManager.java

```diff
@@ -987,7 +986,7 @@ private void initializeServiceMonitors() {
                 m_svcMonitors.put(locator.getServiceName(), monitor);
             });
             if(!monitorFuture.isDone()) {
-                LOG.warn("The monitor {} with class not available.", locator.getServiceName());
+                LOG.warn("The monitor with class {} not available yet, if the feature is installed correctly it will be available latter.", locator.getServiceName());
```

Comment:
```suggestion
                LOG.warn("The monitor with class {} not available yet, if the feature is installed correctly it will be available later.", locator.getServiceName());
```